### PR TITLE
IBX-6618: Updated global variables

### PR DESCRIPTION
**Pull request by juskora**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-6618
| Versions      | 4.6, master
| Edition       | 

Scope of the PR:

- Added variables for product names
- Added variables for tags
- All occurrences of products names replaced with variables
- All occurrences of tags and product versions replaced with variables
- Added note for switching off vale in particular docs sections

This PR is still in a test phase and is aimed and checking whether vale validates it as planned, please do not merge.

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/cdp/cdp_activation.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ Next, you can fill in **Ibexa information** they must match the ones provided in
 
 - **Client Secret** and **Client ID** - are used to authenticate against Webhook endpoint. In the configuration they are taken from environment variables in `.env` file.
 
-- **Segment Group Identifier** - identifier of the segment group in Ibexa DXP. It points to a segment group where all the CDP audiences will be stored.
+- **Segment Group Identifier** - identifier of the segment group in[[= product_name =]]. It points to a segment group where all the CDP audiences will be stored.
```
Comment (MagdalenaZuba) [inline docs/cdp/cdp_activation.md:165]:
```suggestion
- **Segment Group Identifier** - identifier of the segment group in [[= product_name =]]. It points to a segment group where all the CDP audiences will be stored.
```

File: docs/resources/release_process_and_roadmap.md [hunk 2]
```diff
@@ -8,18 +8,18 @@ description: "Ibexa DXP releases new versions periodically in different flavors:
...
-- [[= product_name_content =]] is a multichannel and headless content management system.
+- [[= product_name_headless =]] is a multichannel and headless content management system.
 - [[= product_name_exp =]] is a modern modular Digital Experience Platform to build outstanding customer experiences
 - [[= product_name_com =]] is a commerce-ready B2B DXP designed to digitalize your business from customer awareness to purchase and beyond.
 
-Additionally, [[= product_name =]] also has an open-source version called Ibexa OSS.
-Ibexa OSS is developed by Ibexa together with the open source community.
-The Ibexa OSS code is available on GitHub under the GPLv2 license.
+Additionally, [[= product_name =]] also has an open-source version called [[= product_name_oss =]].
+[[= product_name_oss =]] is developed by [[= product_name_name =]] together with the open source community.
```
Comment (MagdalenaZuba) [inline docs/resources/release_process_and_roadmap.md:16]:
```suggestion
[[= product_name_oss =]] is developed by [[= product_name_base =]] together with the open source community.
```

File: docs/resources/release_process_and_roadmap.md [hunk 3]
```diff
@@ -8,18 +8,18 @@ description: "Ibexa DXP releases new versions periodically in different flavors:
...
+The [[= product_name_oss =]] code is available on GitHub under the GPLv2 license.
 It comes with no commercial support and maintenance services. 
 
 We manage the release of [[= product_name =]] using an agile iterative process and a continuous software development model, which is why we provide two kinds of [[= product_name =]] releases:
 
-- Long Term Support releases (LTS) which are supported by Ibexa for a long period of time.
+- Long Term Support releases (LTS) which are supported by [[= product_name_name =]] for a long period of time.
```
Comment (MagdalenaZuba) [inline docs/resources/release_process_and_roadmap.md:22]:
```suggestion
- Long Term Support releases (LTS) which are supported by [[= product_name_base =]] for a long period of time.
```

File: mkdocs.yml [hunk 4]
```diff
@@ -814,10 +814,16 @@ extra:
       url: '/projects/connect'
   # Global variables
   site_display_name: 'Ibexa Documentation'
+  product_base_name: 'Ibexa'
```
Comment (MagdalenaZuba) [inline mkdocs.yml:817]:
```suggestion
  product_name_base: 'Ibexa'
```
That's how you use it in the doc

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

**Review by julitafalcondusza — APPROVED** [review]

**Pull request merged**